test/jdk/java/util/zip/TestZipSourceDateEpoch.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/sh
```

Comment:
Unless there is a specific reason to use a shell script, it would be better to convert this to java.  We have been trying to reduce the usage of shell scripts

Reply:
I had copied an existing example, I obviously chose a bad example! I can re-write with ProcessBuilder... cheers